cpuset/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
maybe add sig/node label to be added for the changes in this library

Reply:
Done